requirements.txt

```diff
@@ -0,0 +1,9 @@
+numpy~=1.21.2
+pandas~=1.3.3
+matplotlib~=3.4.3
+requests~=2.26.0
+retrying~=1.3.3
+pymongo~=4.1.1
```

Comment:
You need to add a todo in your readme, notifying that user should setup mongodb to run. P.S., Is a database really necessary? If yes, can sqlite do the same job?

Reply:
because we need to get altitude and latitude of different locations using `geoapify` API which is unstructured. It is better to use MongoDB to handle this kinda data.